packages/underscore-redirects/package.json

```diff
@@ -25,7 +25,7 @@
     "build:ci": "astro-scripts build \"src/**/*.ts\"",
     "postbuild": "astro-scripts copy \"src/**/*.js\"",
     "dev": "astro-scripts dev \"src/**/*.ts\"",
-    "test": "mocha --exit --timeout 20000"
+    "test": "astro-scripts test \"test/**/*.test.js\""
   },
   "devDependencies": {
     "@types/chai": "^4.3.10",
```

Comment:
Looks like we can remove the mocha and chai dev dependencies here, which seems to be the reason the CI is failing.

Reply:
on it, thanks for the follow up 👍